src/ngx_http_lua_directive.c

```diff
@@ -1649,7 +1649,8 @@ ngx_http_lua_conf_read_lua_token(ngx_conf_t *cf,
             if (q == NULL) {
                 ngx_conf_log_error(NGX_LOG_EMERG, cf, 0,
                                    "Lua code block missing the closing "
-                                   "long bracket \"%*s\"",
+                                   "long bracket \"%*s\", "
+                                   "probably too long lua code block",
```

Comment:
Better be "(maybe the lua code is too long?)"

Reply:
"The inlined Lua code may be too long"